src/plugins/legacy_export/server/routes/import.ts

```diff
@@ -28,7 +28,7 @@ export const registerImportRoute = (router: IRouter, maxImportPayloadBytes: numb
       validate: {
         body: schema.object({
           objects: schema.arrayOf(schema.recordOf(schema.string(), schema.any())),
-          version: schema.string(),
+          version: schema.maybe(schema.string()),
```

Comment:
There are no unit tests for this file, and I opted not to add one for such a simple one-line change.